lib/gems/pending/appliance_console/database_configuration.rb

```diff
@@ -119,18 +119,26 @@ def ask_for_database_credentials
       self.database = just_ask("name of the database on #{host}", database) unless local?
       self.username = just_ask("username", username) unless local?
       count = 0
+      empty_count = 0
```

Comment:
This seems like it overly complicates things, couldn't we just check if the password is empty and bail if it is?